hw/syn/data/dc.hjson

```diff
@@ -25,6 +25,9 @@
                 "--reppath {build_dir}/REPORTS/ ",
                 "--outdir {build_dir}"]
 
-  // Restrict the maximum message count to 10 in each category
-  max_msg_count: 10
+  // Restrict the maximum message count in each category
+  max_msg_count: 20
+  // Do not sanitize regression mails, but do sanitize the published report
+  sanitize_email_results:   "False"
```

Comment:
Looks like HJson parses `true`, `false` to boolean: https://hjson.github.io/syntax.html

Reply:
cool, let me change that to bools.